test/builtin_dataset_mocks.py

```diff
@@ -10,13 +10,13 @@
 import pickle
 import random
 import xml.etree.ElementTree as ET
-from collections import defaultdict, Counter
```

Comment:
Please revert this as it has nothing to do with the PR. In general, you probably want to disable auto formatters that are not compatible with our code format.

Reply:
This change was actually made by the `ufmt` code formatter, which I used as specified in the contributing guide (see: https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#formatting). Should I still revert it?

Reply:
Yes, please. We need to fix the contribution guide to pin the right versions for the auto formatters. cc @NicolasHug 

@Dbhasin1 This change is made by `usort>=1.*`, but we are currently enforcing

https://github.com/pytorch/vision/blob/efb073682101e1f4f43e1f96a755f1e8ae6958ea/.pre-commit-config.yaml#L19

Would you be up to send a PR that changes the instruction to read `pip install ufmt==1.3.2 black==21.9b0 usort==0.6.4`?

Reply:
sure @pmeier, I can do that